releasenotes/notes/asm-send-waf-metrics-over-telemetry-4208c7412e55cccd.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    ASM: Send WAF metrics over telemetry
```

Comment:
nit: We do not need a release note here. Users do not have access to telemetry data. This is an internal feature.